graph/vertex_cover.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+  @author [Rajat Garg](https://github.com/Rajat379)
+  @file
+
+  Implementation of a vertex cover in a graph.
```

Comment:
Please describe how the algorithm works, a Wikipedia or any other source for algorithm explanation, what the algorithm does, etc..